seonoh/ArchitectureStudy/app/src/main/java/com/example/seonoh/seonohapp/CoinAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.seonoh.seonohapp
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.seonoh.seonohapp.model.UseCoinModel
+
+class CoinAdapter : RecyclerView.Adapter<CoinItemViewHolder>() {
+
+    private val mData = mutableListOf<UseCoinModel>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CoinItemViewHolder {
+
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.coin_item, parent, false)
+        return CoinItemViewHolder(view)
+    }
+
+    override fun getItemCount(): Int = mData!!.size
```

Comment:
notNull 단언자(!!) 생략이 가능합니다
```suggestion
    override fun getItemCount(): Int = mData.size
```

Reply:
네 수정했습니다. 감사합니다!

Reply:
@seonoh 
반영 커밋 남겨주세요

Reply:
d2667c66ab59c847c11c8e48da83cc9eac5acef3